node/src/Pos/Client/Txp/History.hs

```diff
@@ -216,11 +217,11 @@ class (Monad m, SscHelpersClass ssc) => MonadTxHistory ssc m | m -> ssc where
 
     default getLocalHistory
         :: (MonadTrans t, MonadTxHistory ssc m', t m' ~ m)
-        => [Address] -> m (Map TxId TxHistoryEntry)
-    getLocalHistory = lift . getLocalHistory
+        => MemPoolSnapshot -> [Address] -> m (Map TxId TxHistoryEntry)
+    getLocalHistory m = lift . getLocalHistory m
```

Comment:
Might as well be consistent in the short var name we pick. `mps` seems fine.

Reply:
Oh totally, must be a typo, `m` is such a terrible one 😀